time_tracker_api/time_entries/time_entries_namespace.py

```diff
@@ -267,6 +268,24 @@ def get(self):
         return time_entries_dao.get_lastest_entries_by_project(conditions={})
 
 
+@ns.route('/feature-toggles')
+class FeaturesToggles(Resource):
```

Comment:
Not sure if this piece of code (lines 271 to 288)should be included.  You already tested, your code works. I consider that we could remove this and put this whole example in the JIRA issue(as an example of how to use the current implementation).

e.g.:
```
ftm = FeatureToggleManager('<my-feature-flag>')
is_enabled = ftm.is_toggle_enabled_for_user()
if is_enabled:
    # My logic if toggle is enabled....
else:
   # My logic if toggle is disabled....
```

Reply:
Ok, we have moved this example code to JIRA.